packages/calcite-components/src/components/color-picker/color-picker.e2e.ts

```diff
@@ -2345,4 +2355,274 @@ describe("calcite-color-picker", () => {
+          },
+          {
+            shadowSelector: `.${CSS.saveColor}`,
+            targetProp: "--calcite-button-background-color",
+            state: { press: { attribute: "class", value: CSS.saveColor } },
+          },
+        ],
+        "--calcite-color-picker-button-background-color-focus": [
+          {
+            shadowSelector: `.${CSS.deleteColor}`,
+            targetProp: "--calcite-button-background-color",
+            state: { focus: { attribute: "class", value: CSS.deleteColor } },
+          },
+          {
+            shadowSelector: `.${CSS.saveColor}`,
+            targetProp: "--calcite-button-background-color",
+            state: { focus: { attribute: "class", value: CSS.saveColor } },
+          },
+        ],
+        "--calcite-color-picker-button-background-color-hover": [
+          {
+            shadowSelector: `.${CSS.deleteColor}`,
+            targetProp: "--calcite-button-background-color",
+            state: "hover",
+          },
+          {
+            shadowSelector: `.${CSS.saveColor}`,
+            targetProp: "--calcite-button-background-color",
+            state: "hover",
+          },
+        ],
+        "--calcite-color-picker-button-icon-color": [
+          {
+            shadowSelector: `.${CSS.deleteColor}`,
+            targetProp: "--calcite-button-icon-color",
+          },
+          {
+            shadowSelector: `.${CSS.saveColor}`,
+            targetProp: "--calcite-button-icon-color",
+          },
+        ],
+        "--calcite-color-picker-button-icon-color-active": [
+          {
+            shadowSelector: `.${CSS.deleteColor}`,
+            targetProp: "--calcite-button-icon-color",
+            state: { press: { attribute: "class", value: CSS.deleteColor } },
+          },
+          {
+            shadowSelector: `.${CSS.saveColor}`,
+            targetProp: "--calcite-button-icon-color",
+            state: { press: { attribute: "class", value: CSS.saveColor } },
+          },
+        ],
+        "--calcite-color-picker-button-icon-color-focus": [
+          {
+            shadowSelector: `.${CSS.deleteColor}`,
+            targetProp: "--calcite-button-icon-color",
+            state: { focus: { attribute: "class", value: CSS.deleteColor } },
+          },
+          {
+            shadowSelector: `.${CSS.saveColor}`,
+            targetProp: "--calcite-button-icon-color",
+            state: { focus: { attribute: "class", value: CSS.saveColor } },
+          },
+        ],
+        "--calcite-color-picker-button-icon-color-hover": [
+          {
+            shadowSelector: `.${CSS.deleteColor}`,
+            targetProp: "--calcite-button-icon-color",
+            state: "hover",
+          },
+          {
+            shadowSelector: `.${CSS.saveColor}`,
+            targetProp: "--calcite-button-icon-color",
+            state: "hover",
+          },
+        ],
+        "--calcite-color-picker-button-text-color": [
```

Comment:
--calcite-color-picker-text-color and --calcite-color-picker-button-text-color have the same color by default. I think we can simplify this to 
`--calcite-color-picker-text-color` + the various states